templates/ovncontroller/bin/start-vswitchd.sh

```diff
@@ -30,7 +30,14 @@ ovs-vsctl --no-wait set open_vswitch . other_config:flow-restore-wait=true
 
 # It's safe to start vswitchd now. Do it.
 # --detach to allow the execution to continue to restoring the flows.
-/usr/sbin/ovs-vswitchd --pidfile --mlockall --detach
+$EXECUTE_VSWITCHD_SCRIPT > /dev/stdout 2>&1 &
```

Comment:
why we need additonal bash script for just this command? running /usr/sbin/ovs-vswitchd --pidfile --mlockall > /dev/stdout 2>&1 & have some issues?